.github/workflows/codecov_ats.yml

```diff
@@ -80,13 +83,36 @@ jobs:
           env:
             CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
             CODECOV_STATIC_TOKEN: ${{ secrets.CODECOV_STATIC_TOKEN }}
-        # Run label analysis IN DRY MODE (no tests will actually run)
+        # Run label analysis in dry mode to get the list of tests to run
+        # The base commit will be the parent commit (apparently commits on master don't exist in codecov)
         - name: Label Analysis
+          id: label_analysis
           continue-on-error: true
           run: |
-            BASE_COMMIT=$(git merge-base ${{ github.sha }}^ origin/master)
+            set +e
```

Comment:
Don't `set +e`, please. The equivalent python would be wrapping the entire file with `try: ... except: pass`.
Instead, if there's particular commands you want to ignore the exit code, append `|| true` to them.
A comment about why the error is expected would be nice too.